plugins/janus_videoroom.c

```diff
@@ -441,6 +442,11 @@ typedef struct janus_videoroom_rtp_relay_packet {
 
 
 /* Freeing stuff */
+static void janus_videoroom_subscriber_dereference(janus_videoroom_subscriber *s) {
+	if(s)
```

Comment:
I checked the source for [GHashTable](https://github.com/GNOME/glib/blob/8c9be18bd3dff42a2352c2dac6b8f17b243d2980/glib/ghash.c#L903) and there is nothing that prevents `NULL` values.  Still, we could remove the test if we are sure that we would never insert a `NULL` publisher into the participants table.